plugins/wiggle/src/BigWigAdapter/BigWigAdapter.ts

```diff
@@ -61,7 +61,7 @@ export default class BigWigAdapter extends BaseFeatureDataAdapter {
     return (h.refsByNumber[refId] || { name: undefined }).name
   }
 
-  public async getGlobalStats(opts?: BaseOptions) {
+  public async getGlobalStats(region?: Region, opts?: BaseOptions) {
```

Comment:
why does global stats get a region argument?

Reply:
since the baseadapter estimate global stats needs a region for it's estimation and the baseadapter getGlobalStats calls estimateGlobalStats